src/java/org/apache/cassandra/tools/LoaderOptions.java

```diff
@@ -22,25 +22,37 @@
 
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
-import java.net.*;
+import java.net.InetAddress;
```

Comment:
I am not against the changes. Those are good and more in line with the code style, but unrelated changes of the patch.

Reply:
should I revert these changes?